include/drm/drm_gem.h

```diff
@@ -119,6 +119,11 @@ struct drm_gem_object {
 	 * simply leave it as NULL.
 	 */
 	struct dma_buf_attachment *import_attach;
+
+	/*
+	 * is backed by scattered pages
+	 */
+	bool is_scattered;
```

Comment:
Agree but I don't see how I can do this otherwise. I can drop this once I unify CMA and scattered approach but right now I have no other mean to differentiate between the two from inside the hooks.